wal/wal.go

```diff
@@ -429,7 +429,6 @@ func (w *WAL) flushPage(clear bool) error {
 	// No more data will fit into the page. Enqueue and clear it.
```

Comment:
Comment is not strictly true -> clear bool also matters (:

Reply:
updated